corehq/apps/app_manager/static/app_manager/js/details/column.js

```diff
@@ -53,7 +53,9 @@ hqDefine("app_manager/js/details/column", function () {
         self.tileRowMax = ko.observable(7);
         self.tileColumnMax = ko.observable(13);
         self.tileRowStart = ko.observable(self.original.grid_y || 1);
-        self.tileRowOptions = [""].concat(_.range(1, self.tileRowMax()));
+        self.tileRowOptions = ko.computed(function () {
```

Comment:
I was expecting to see a reference to this change from `tileRowOptions` to `tileRowOptions()`, but it looks like it's only used in a template, and knockout magic makes it so you don't need to call it there.